ngraph/test/visitors/op/unary_ops.hpp

```diff
@@ -0,0 +1,70 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//  Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
```

Comment:
This is old header, please use new, short version.

```
// Copyright (C) 2021 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//
```

Reply:
Thanks! Fixed in the commit.